### DISCUSSION TITLE
Want to accelerate the example

### DISCUSSION THREAD

**Discussion by SQDSDA** [julianspaeth/random-survival-forest#16] [Q&A]
When I run this example in device of kaggle, it need a lot of time to get the results. My data dimensions are 499*20, and I want to get some tricks to accelerate that. Can you give  me some suggestions? thanks

**Comment by julianspaeth** [comment]
Unfortunately, this pure python implementation is not very performant. The bottleneck is the splitting rule - In the past, I used the [fastlogranktest](https://github.com/compsysmed/fastlogranktest) package, but this was not maintained anymore - I will have a look at it and try to speed it up a little.

**Reply by julianspaeth** [reply]
Better late than never - the performance should be better now
